includes/woopay/class-woopay-utilities.php

```diff
@@ -23,6 +23,7 @@ class WooPay_Utilities {
 	const AVAILABLE_COUNTRIES_KEY            = 'woocommerce_woocommerce_payments_woopay_available_countries';
 	const AVAILABLE_COUNTRIES_LAST_CHECK_KEY = 'woocommerce_woocommerce_payments_woopay_available_countries_last_check';
 	const AVAILABLE_COUNTRIES_DEFAULT        = '["US"]';
+	const SUPPORTED_EXTENSIONS               = [ 'woocommerce-points-and-rewards/woocommerce-points-and-rewards.php', 'woocommerce-gift-cards/woocommerce-gift-cards.php' ];
```

Comment:
This list will be updated on WooPay after https://github.com/Automattic/woocommerce-payments/pull/6562.